app/views/payments/new.html.haml

```diff
@@ -0,0 +1,14 @@
+.container
+  .row
+    .col-xs-6.col-xs-offset-3
+      %h1
+        Buy tickets for
+        = @total_amount_to_pay.symbol
```

Comment:
Okay, would it make more sense to use `Money#format` instead of `.symbol` and `humanized_money` together?